kratos/includes/kratos_parameters.h

```diff
@@ -307,6 +392,41 @@ class Parameters
 //         mpvalue->SetString(rapidjson::StringRef(value.c_str()));
 //        mpvalue->SetString(value.c_str(), value.length());
     }
+    void SetVector(const Vector& vec) // Riccardo how to pass this?
+    {
+        // Riccardo there s also the "SetArrayRaw" method...would you prefer this one?
```

Comment:
To be honest I don't know. Look into the rapidjson documentation, or ask in the relevant list (I would have to do the same)